NeoML/include/NeoML/Dnn/DnnSimpleAPI.h

```diff
@@ -0,0 +1,512 @@
+/* Copyright © 2017-2020 ABBYY Production LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--------------------------------------------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <NeoML/Dnn/DnnLambdaHolder.h>
+#include <NeoML/NeoMLDefs.h>
+#include <NeoML/Dnn/Dnn.h>
+#include <NeoML/Dnn/Layers/DropoutLayer.h>

```

Comment:
Almost all of Layers' headers can be moved to cpp (and replaced here with forward declarations).

The only exception is the case, when enum is needed from the header like `CSoftmaxLayer::TNormalizationArea`  or `CPositionalEmbeddingLayer::TPositionalEmbeddingType`

Reply:
**on hold** I'm not so sure if my comment above is actually a good idea...

Reply:
Yes, but this is simple API for fast prototyping.